notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1032 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    tp = lk.read(tp_path, quality_bitmask="none", flux_column="sap_flux")
</pre><p> </p><p>Add note that we don't want a corrected version of the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>added</p>